examples/exempt_custom_fees/main.go

```diff
@@ -166,6 +142,27 @@ func main() {
 	 *
 	 * Collector 0.0.B sends 10_000 units of the token to 0.0.A.
 	 */
+
+	// First we transfer the amount from treasury account to second account
+	treasuryTokenTransderTransaction, err := hedera.NewTransferTransaction().
+		AddTokenTransfer(tokenId, id, -10000).AddTokenTransfer(tokenId, secondAccountId, 10000).
+		FreezeWith(client)
+	if err != nil {
+		fmt.Println(err)
+	}
+	treasuryTokenTransderTransactionSign := treasuryTokenTransderTransaction.Sign(key)
+
+	treasuryTokenTransferSubmit, err := treasuryTokenTransderTransactionSign.Execute(client)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** We can use directly `treasuryTokenTransderTransaction.Sign(key).Execute(client)` instead of introducing `treasuryTokenTransderTransactionSign` variable.‏